.github/ISSUE_TEMPLATE/feature-request.md

```diff
@@ -0,0 +1,26 @@
+---
+name: Feature request
+about: Suggest an idea for this project
+title: ''
+labels: ''
+assignees: ''
+
+---
+
+**Is your feature request related to a problem? Please describe.**
+A clear and concise description of what the problem is. Ex. I'm always frustrated when [...]
+
+**Describe the solution you'd like**
+A clear and concise description of what you want to happen.
+
+**Describe alternatives you've considered**
```

Comment:
I'm afraid those questions above are not easy to answer. Would you mind just copying https://github.com/tensorflow/tensorflow/blob/master/.github/ISSUE_TEMPLATE/30-feature-request.md here?

and ask more information like:
```
System information
    TensorFlow version (you are using):
    TensorFlow Addons version:
    Is it in the tf.contrib (and where):
    Are you willing to contribute it (Yes/No):
```

Reply:
Copied